controllers/cloud.redhat.com/clowdapp_controller.go

```diff
@@ -157,7 +157,7 @@ func (r *ClowdAppReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	return ctrl.Result{Requeue: requeue}, nil
 }
 
-// SetupWithManager sets up wi
+// SetupWithManager sets up with Manager
```

Comment:
haha

Reply:
Yeh I keep meaning to fix that one..........